packages/tocco-ui/src/Table/StyledTable.js

```diff
@@ -53,6 +61,15 @@ export const StyledTableHeaderCell = styled.th`
     }
     `
   }
+  &#header-cell-navigation-column {
```

Comment:
The navigation cell is created and maintained in the entity-list app. Introducing a id selector in the generic tocco-ui table will compromise our goal so separate these app. I think we have to extend the column definition. Can you think of a generic attribute for this column to style accordingly? 
i must have missed this in the last review